modules/about-certificate-management.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-a-custom-pki.adoc
+
+[id="about-certificate-management_{context}"]
+= About certificate management
+
+Once a custom certificate is set, the network operator merges the user-provided
+and system CA certificates into a single bundle and injects the merged bundle
+into the operator requesting the trust bundle injection.
+
+Operators request this injection by creating an empty ConfigMap with the
+following label:
+
+[source,yaml]
+----
+config.openshift.io/inject-trusted-cabundle="true"
+----
+
+The operator mounts this ConfigMap into the container's local trust store.
```

Comment:
@danehans perhaps we should list the supported operators here? What would those be?

Reply:
I think maintaining the list will be very difficult. Any operator that makes a call external to the cluster requires proxy support. Note that ingress operator will be on this list when https://jira.coreos.com/browse/NE-182 is done.

Reply:
Good point; I'll leave the list out due to maintainability issues. Just to confirm, the proxy API is only valid for 4.2+, correct?